src/components/Goals/BikeGoal/BikeGoalAlertSwitcher.jsx

```diff
@@ -24,7 +24,7 @@ const BikeGoalAlertSwitcher = ({ className }) => {
   const { isMobile } = useBreakpoints()
   const classes = useStyles()
 
-  const { isLoading, value = true, save } = useSettings('bikeGoal.showAlert')
+  const { isLoading, value, save } = useSettings('bikeGoal.showAlert')
```

Comment:
We will have the same warning here, and also an inconsistency by default, where the blue alert would be shown but this switch would be unchecked.